### ISSUE TITLE
res2net50-v1b的训练策略

### ISSUE THREAD

**Issue by zp5070** [Res2Net/Res2Net-PretrainedModels#71]
您好，我现在在复现res2net在imagenet数据集上的分类精度，我参考了论文，res2net50的训练策略如下：
scheduler: 'step_decay'
lr: 0.1
warmup_epochs: 0
decay_rate: 0.1
decay_epochs: 30
epoch_size: 100
res2net精度可以达到top1: 78.00%，和论文是相符的；
在此基础上添加了cutmix数据增强策略，epoch_size变为300，每75epoch学习率下降为原来的0.1，来训练res2net50-v1b；res2net50-v1b实测精度只有79.80%，与论文相比还差0.5%。因为代码中没有提供训练的参数，可能在某些地方没对齐，想问一下能不能分享一下res2net50-v1b的训练策略或者训练参数。谢谢。

**Comment by gasvn** [comment]
res2net-v1b使用了cutmix，也同时采用了cutmix对应的训练策略。可以参考这个代码实现 https://github.com/ildoonet/cutmix
另外res2net的训练技术相对较老，如果想获得更好的性能可以直接使用最近较新的训练策略, 例如 https://pytorch.org/blog/how-to-train-state-of-the-art-models-using-torchvision-latest-primitives/

**Comment by zp5070** [comment]
好的，谢谢啦
